doc/guides/test/sanitycheck.rst

```diff
@@ -201,8 +201,10 @@ required for best test coverage for this specific board:
 
 identifier:
   A string that matches how the board is defined in the build system. This same
-  string is used when building, for example when calling 'cmake'::
+  string is used when building, for example when calling ``west build`` or
+  ``cmake``::
 
+  # west build -b quark_d2000_crb
```

Comment:
Maybe separate these commands with an or? It might look like you need to run both in order.

```
or ``cmake``::

   # west build ...

or::

  # cmake ...
```

While you're here, is `#` the right character to be using here? I would have expected either nothing or `$`.